packages/replayer/src/replayer.ts

```diff
@@ -129,17 +128,10 @@ export const replayEvents: ReplayEventsFn = async (options) => {
   // Set-up network stubbing if required
   if (networkStubbing) {
     // eslint-disable-next-line @typescript-eslint/no-var-requires
-    const networkStubbingModule = require(dependencies.nodeNetworkStubbing
+    const networkStubbingExports = require(dependencies.nodeNetworkStubbing
```

Comment:
Nitpick: why rename it? I see it as an ES or CommonJS module since it is the result of calling `require`.

Reply:
It wasn't really intentional rename / I don't have any real preference between the two names. I refactored it out into a function and then back again when I changed my mind, and so ended up with a different name.